src/ACEtk/interpretation/EL03/OccupationNumber.hpp

```diff
@@ -0,0 +1,12 @@
+class OccupationNumber {
+  const Table& table;
+
+public:
+  OccupationNumber( const Table& table ) : table( table ){}
+    
+  auto values() const {
+    const auto length = this->table.data.NXS( 11 );
+    const auto start  = this->table.data.JXS( 11 );
+    return this->table.data.XSS( start, length );
```

Comment:
negative occupation == conductor, right?

Maybe we should add a transform for `abs` to avoid that nonsense and add a `conductive` method